ctapipe/containers.py

```diff
@@ -523,10 +523,27 @@ class ReconstructedGeometryContainer(Container):
         nan * u.m, "reconstructed x coordinate of the core position", unit=u.m
     )
     core_y = Field(
-        nan * u.m, "reconstructed y coordinate of the core position", unit=u.m
+        nan * u.m,
+        "reconstructed y coordinate of the core position",
+        unit=u.m
     )
     core_uncert = Field(
-        nan * u.m, "uncertainty of the reconstructed core position", unit=u.m
+        nan * u.m,
+        "uncertainty of the reconstructed core position in the ground frame",
+        unit=u.m
+    )
+    core_tilted_x = Field(
+        nan * u.m, "reconstructed x coordinate of the core position", unit=u.m
+    )
+    core_tilted_y = Field(
+        nan * u.m,
+        "reconstructed y coordinate of the core position",
+        unit=u.m
+    )
+    core_uncert_tilted = Field(
```

Comment:
Why only one uncertainty? Should we have one for each x and y?

Reply:
doesn't matter much at this point, since it is not even computed, but yes it should be for each dimension - should I fix it here also for the GroundFrame?

Reply:
it was originally only one for simplicity ( just to have an average number, like a spread of the pair-wise impact positions), but of course there are cases where it could be asymmetric, so it wouldn't hurt to make it 2d.  Also, it's already x/y for the alt/az coordinate, so this should be the same.

Reply:
Just remember to bump the data model number in `datawriter.py` as well (since we both added new variables and changed existing ones, if you decide to also modify the uncertainty here)

Reply:
should it be 1.2.1 or 1.3.0?

Reply:
1.3 is fine since you are adding new columns.  Technically we should change the major number if you rename a column, however, but let's think about that before the release.

Reply:
done